x-pack/solutions/observability/plugins/apm/ftr_e2e/cypress/e2e/service_overview/otel_edot_service_overview_and_transactions.cy.ts

```diff
@@ -94,12 +100,40 @@ describe('Service Overview', () => {
     });
 
     it('navigates to transaction detail page', () => {
-      cy.visitKibana(baseUrl);
-      cy.contains('Transactions').click();
-
       cy.contains('a', 'oteldemo.AdServiceEdotSynth/GetAds').click();
       cy.contains('h5', 'oteldemo.AdServiceEdotSynth/GetAds');
     });
+    it('shows transaction summary', () => {
```

Comment:
Why do we add those here? Weren't they already there, or was it a merge conflict 🤔